be/db.go

```diff
@@ -12,9 +12,18 @@ import (
 )
 
 var (
-	mysqlAddress = flag.String("mysql_address", "server:dev_pass@tcp(cleanupdb:3306)/cleanapp", "MySQL address string")
+	mysqlPassword = flag.String("mysql_password", "secret", "MySQL password.")
+	mysqlHost = flag.String("mysql_host", "cleanupdb", "MySQL host.")
+	mysqlPort = flag.String("mysql_port", "3306", "MySQL port.")
+	mysqlDb = flag.String("mysql_db", "cleanapp", "MySQL database to use.")
 )
 
+func mysqlAddress() string {
+	db := fmt.Sprintf("server:%s@tcp(%s:%s)/%s", *mysqlPassword, *mysqlHost, *mysqlPort, *mysqlDb)
+	// log.Printf("DB connect string: %q", db) // Troubleshooting only. Exposes password.
```

Comment:
Let's remove a commented code.

Reply:
It would be useful to me onthe second phase, but ok, done.